src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -191,9 +191,10 @@ class DocumentController(
       throw WrongDocumentTypeException(generateDocumentRequest.category)
 
     return when (generateDocumentRequest.category) {
-      RECALL_NOTIFICATION -> recallNotificationService.generateAndStorePdf(recallId, currentUserUuid, generateDocumentRequest.details).map { it.first }
+      RECALL_NOTIFICATION -> recallNotificationService.generateAndStorePdf(recallId, currentUserUuid, generateDocumentRequest.details)
       REVOCATION_ORDER -> revocationOrderService.generateAndStorePdf(recallId, currentUserUuid, generateDocumentRequest.details)
       REASONS_FOR_RECALL -> reasonsForRecallService.generateAndStorePdf(recallId, currentUserUuid, generateDocumentRequest.details)
+      DocumentCategory.DOSSIER -> dossierService.generateAndStorePdf(recallId, currentUserUuid, generateDocumentRequest.details)
```

Comment:
Cosmetic, but add import?

Reply:
Ha, weird. I tried to do it but it didn't offer choice. I assumed there was another x.DOSSIER somewhere. Seems not. Will tidy up.